gov/politeia/proposals.go

```diff
@@ -267,23 +269,33 @@ func (db *ProposalDB) updateInProgressProposals() (int, error) {
 	for _, val := range inProgress {
 		proposal, err := piclient.RetrieveProposalByToken(db.client, db.APIURLpath,
 			val.Censorship.Token)
+		// Do not update if:
+		// 1. piclient.RetrieveProposalByToken returned an error
 		if err != nil {
+			return 0, fmt.Errorf("RetrieveProposalByToken failed: %v ", err)
```

Comment:
Did you mean to put this return?  The log and continue were just added recently

Reply:
Nope. This was an error I made while transferring the code. Fixing ASAP.